L1Trigger/Phase2L1ParticleFlow/src/puppi/linpuppi_ref.cpp

```diff
@@ -13,8 +13,11 @@
 #include "FWCore/ParameterSet/interface/allowedValues.h"
 #include "FWCore/Utilities/interface/transform.h"
 #include "FWCore/Utilities/interface/Exception.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
 #endif
 
+#include "L1Trigger/Phase2L1ParticleFlow/interface/NNVtxAssoc.h"
```

Comment:
Somehow the hole NNVtx association will need to be protected for the puppi emulator to compile in vivado, isn't it?